doc/install-dependencies.md

```diff
@@ -65,6 +66,21 @@ sudo yum install -y etcd openresty curl git gcc luarocks lua-devel
 sudo service etcd start
 ```
 
+Fedora 31 & 32
+==============
+
+```shell
+# add OpenResty source
+sudo yum install yum-utils
+sudo yum-config-manager --add-repo https://openresty.org/package/fedora/openresty.repo
+
+# install OpenResty, etcd and some compilation tools
+sudo yum install -y etcd openresty curl git gcc luarocks lua-devel
+
+# start etcd server
+sudo service etcd start
```

Comment:
we need to confirm we enable the v2 protocol.

`etcd --enable-v2` is the simplest way.

Reply:
@membphis good catch and done. I would prefer to use this simplest way rather than changing systemd service file as the `v3` should be also supported. 😄  Much appreciated.